packages/shared/components/popups/ConfirmationPopup.svelte

```diff
@@ -13,24 +13,32 @@
     export let onCancel: () => void = undefined
 
     function confirmClick(): void {
-        onConfirm()
+        if (onConfirm) {
+            onConfirm()
+        }
+        closePopup()
     }
 
     function cancelClick(): void {
         if (onCancel) {
             onCancel()
-        } else {
-            closePopup()
         }
+        closePopup()
```

Comment:
I think this change here collides with the current behavior of async activities `packages/shared/components/popups/ActivityDetailsPopup.svelte#59`. Async tx > open detail > click reject > cancel > popups should be kept open but on the previous step